releases/release-5.4.3.md

```diff
@@ -0,0 +1,143 @@
+    - (dup) Fix a bug that Regions might be overlapped if Raftstore is busy [#13160](https://github.com/tikv/tikv/issues/13160)
+
++ PD **owner: @nolouch**
+
+    - Fix a bug that could not handle dashboard proxy requests correctly [#5321](https://github.com/tikv/pd/issues/5321)
+    - (dup) Fix the issue that a removed tombstone store appears again after the PD leader transfer ​​[#4941](https://github.com/tikv/pd/issues/4941)
+    - (dup) Fix the bug that the TiFlash learner replica might not be created [#5401](https://github.com/tikv/pd/issues/5401)
+
++ TiFlash
+
+    <!--compute **owner: @zanmato1984**-->
+
+    - (dup) Fix the issue that the `format` function might return a `Data truncated` error [#4891](https://github.com/pingcap/tiflash/issues/4891)
+    - (dup) Fix a bug that TiFlash might crash due to an error in parallel aggregation [#5356](https://github.com/pingcap/tiflash/issues/5356)
+
+    <!--storage **owner: @flowbehappy**-->
+
+    - (dup) Fix the panic issue that occurs after creating the primary index with a column containing the `NULL` value [#5859](https://github.com/pingcap/tiflash/issues/5859)
+
++ Tools
+
+    + TiDB Lightning **owner: @niubell**
+
+        - Fix the issue of BigInt auto_increment column out of range error [#27397](https://github.com/pingcap/tidb/issues/27937)
+        - (dup) Fix the issue that de-duplication might cause TiDB Lightning to panic in extreme cases [#34163](https://github.com/pingcap/tidb/issues/34163)
+        - (dup) Fix the issue that TiDB Lightning does not support columns starting with slash, number, or non-ascii characters in Parquet files [#36980](https://github.com/pingcap/tidb/issues/36980)
+        - (dup) Fix the issue that TiDB Lightning fails to connect to TiDB when TiDB uses an IPv6 host [#35880](https://github.com/pingcap/tidb/issues/35880)
+
+    + TiDB Data Migration (DM) **owner: @niubell**
+
+        - (dup) Fix the issue that DM Worker might get stuck when getting DB Conn [#3733](https://github.com/pingcap/tiflow/issues/3733)
+        - (dup) Fix the issue that DM reports the `Specified key was too long` error [#5315](https://github.com/pingcap/tiflow/issues/5315)
+        - (dup) Fix a bug that latin1 data might be corrupt during replication [#7028](https://github.com/pingcap/tiflow/issues/7028)
+        - (dup) Fix the issue that DM fails to start when TiDB uses an IPv6 host [#6249](https://github.com/pingcap/tiflow/issues/6249)
+        - (dup) Fix the issue of possible data race in `query-status` [#4811](https://github.com/pingcap/tiflow/issues/4811)
+        - (dup) Fix goroutine leak when relay meets an error [#6193](https://github.com/pingcap/tiflow/issues/6193)
+
+    + TiCDC **owner: @nongfushanquan**
+
+        - (dup) Fix the TiCDC panic issue when you set `enable-old-value = false` [#6198](https://github.com/pingcap/tiflow/issues/6198)
```

Comment:
- (dup) reduce the cdc goroutines' number when initializing a capture with lots of regions and tikv.cdc.min-ts-interval = 200ms [#5610](https://github.com/pingcap/tiflow/issues/5610)

Reply:
> - (dup) Reduce performance overhead caused by runtime context switching in multi-Region scenarios [#5610](https://github.com/pingcap/tiflow/issues/5610)

This note has existed in [L46](https://github.com/pingcap/docs/pull/10581/files#diff-9bf75c89d4e623f4fd2e4e442fc2c2de31d0eb3843a6b5368476531584749c2fR46).